### PR TITLE
Styling

### PR DESCRIPTION
**Pull request by samuelstevens**
Screen Shot 2020-10-20 at 6 09 42 PM

Updated styling!

Open to criticism and advice. I think we should merge this into master and then everyone can play with it.

### CODE REVIEW

**Comment by samuelstevens** [comment]
This branch also adds Alex's subscribe to playlist feature

File: watsong/static/images/dial-face.svg [hunk 1]
```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE svg PUBLIC "-//W3C//DTD SVG 1.1//EN" "http://www.w3.org/Graphics/SVG/1.1/DTD/svg11.dtd">
+<svg version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0" y="0" width="179.85" height="179.85" viewBox="0, 0, 179.85, 179.85">
+  <g id="Dial" transform="translate(-429.127, -269.176)">
+    <g>
+      <path d="M606.914,340.037 C617.442,388.562 586.64,436.434 538.116,446.962 C489.591,457.491 441.719,426.689 431.19,378.165 C420.662,329.64 451.464,281.768 499.988,271.239 C548.513,260.711 596.385,291.512 606.914,340.037 z" fill="#686868"/>
```
Comment (Alex7Li) [inline watsong/static/images/dial-face.svg:6]: Just wondering, what does all this code mean

Comment (Alex7Li) [reply watsong/static/images/dial-face.svg:6]: And in the other files

File: watsong/static/js/webapp.js [hunk 2]
```diff
@@ -33,67 +33,67 @@ function getCoord(event) {
  * @param {string} dialName is the dial name to be sent to the backend
  */
 function initializeDial(elem, dialName) {
-    var currentAngle = 0, prevAngle = 0, initialAngle = 0, center = getCenter(elem);
+  var currentAngle = 0, prevAngle = 0, initialAngle = 0, center = getCenter(elem);
```
Comment (Alex7Li) [inline watsong/static/js/webapp.js:36]: let is better style than var, which is mainly used in older js code. https://google.github.io/styleguide/jsguide.html#features-use-const-and-let

**Review by Alex7Li — CHANGES_REQUESTED** [review]
Wow, that looks great! I really love the theme and font. Thanks for working on all of this! I can't comment on the css code itself because I don't understand it too well but it looks good on several sizes on screen on my device so i think it is good css code :P 

And I had a few things that came to mind while I was looking at the code, most aren't related to this PR but they are related to the front end
Things that we could still work on (in another PR or never):
When I hover over a autosuggested value on the search bar, the styling drops.
The rotating dials should have labels on them
The scroll bar is a bit ugly
Some songs are loaded when you first enter the page, before searching for anything

**Comment by AnthonyFortuna** [comment]
Looks great. I understand Alex's issues but for the sake of this current presentation I believe we should be okay

**Review by Alex7Li — APPROVED** [review]
Merge for now

File: watsong/static/images/dial-face.svg (hunk 1 above)
Comment (samuelstevens) [reply watsong/static/images/dial-face.svg:6]: It's a [vectorized image file](https://en.wikipedia.org/wiki/Scalable_Vector_Graphics).

**Pull request merged**